tests/integration_test/history_client_test.py

```diff
@@ -483,146 +583,118 @@ def test_engaged_user_range_with_kairon_client(mock_auth, mock_mongo_processor):
 
 
 @responses.activate
-def test_new_user_range_with_kairon_client(mock_auth, mock_mongo_processor):
-    responses.add(
-        responses.GET,
-        f"https://localhost:8083/api/history/{pytest.bot}/trends/users/new",
-        status=200,
-        match=[responses.json_params_matcher({'month': 6})],
-        json={"data": {'new_user_range': {1: 25, 2: 24, 3: 28, 4: 26, 5: 20, 6: 25}}}
-    )
-
-    response = client.get(
-        f"/api/history/{pytest.bot}/metrics/trend/user/new",
-        headers={"Authorization": pytest.token_type + " " + pytest.access_token},
-    )
-
-    actual = response.json()
-    assert actual["error_code"] == 0
-    assert actual["data"]['new_user_range'] == {'1': 25, '2': 24, '3': 28, '4': 26, '5': 20, '6': 25}
-    assert actual["message"] is None
-    assert actual["success"]
-
-
-@responses.activate
-def test_successful_conversation_range_with_kairon_client(mock_auth, mock_mongo_processor):
-    responses.add(
-        responses.GET,
-        f"https://localhost:8083/api/history/{pytest.bot}/trends/conversations/success",
-        status=200,
-        match=[responses.json_params_matcher({'month': 6, 'action_fallback': 'action_default_fallback',
-                                              'nlu_fallback': 'utter_please_rephrase'})],
-        json={"data": {"successful_sessions": {1: 25, 2: 24, 3: 28, 4: 26, 5: 20, 6: 25}}}
-    )
-
-    response = client.get(
-        f"/api/history/{pytest.bot}/metrics/trend/conversation/success",
-        headers={"Authorization": pytest.token_type + " " + pytest.access_token},
-    )
-
-    actual = response.json()
-    assert actual["error_code"] == 0
-    assert actual["data"]['successful_sessions'] == {'1': 25, '2': 24, '3': 28, '4': 26, '5': 20, '6': 25}
-    assert actual["message"] is None
-    assert actual["success"]
-
-
-@responses.activate
-def test_user_retention_range_with_kairon_client(mock_auth, mock_mongo_processor):
+def test_engaged_user_range_with_value_with_kairon_client(mock_auth, mock_mongo_processor):
+    from_date = datetime.date.today() - datetime.timedelta(30)
+    to_date = datetime.date.today()
     responses.add(
         responses.GET,
-        f"https://localhost:8083/api/history/{pytest.bot}/trends/users/retention",
```

Comment:
why endpoint changed in rest of the test cases?
the test method names have also been changed. why?